src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --colorBlye: #00acdc;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("/fonts/Roboto-Medium.ttf");
+  font-weight: 500;
+  font-style: normal;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 16px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__link {
+  margin: 10px 0;
+}
+
+.nav__list {
+  margin: 19px 0;
```

Comment:
no needs to set margin to a list. You need:
1) add align-items: center to your list (if there is display: flex)
2) add to your links display: block and padding: 23px 0;